redirects-transition.conf

```diff
@@ -189,13 +200,17 @@ rewrite ^/law/cfr/ej_compilation/2016/notice2016-01.pdf https://www.fec.gov/reso
 rewrite ^/law/cfr/ej_compilation/2015/notice2015-01.pdf https://www.fec.gov/resources/cms-content/documents/fedreg_notice_2015-01.pdf redirect;
 
 # law/cfr/ej_compilation/2014/ redirects
+rewrite ^/law/cfr/ej_compilation/2014/notice2014-01.pdf	https://sers.fec.gov/fosers/showpdf.htm?docid=304204 redirect;
```

Comment:
Check spacing between URLs please

Reply:
@patphongs Thank you for helping us fix all of these. I made the fixes we discussed plus the non-E&J ones you flagged above. I think it's ready for you again. Thanks!